soc/arm/microchip_mec/mec172x/soc.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __MEC_SOC_H
+#define __MEC_SOC_H
+
+#define SYSCLK_DEFAULT_IOSC_HZ MHZ(96)
+
+#ifndef _ASMLANGUAGE
```

Comment:
Is this intended to exclude all the includes below, as well as the __FPU_PRESENT macro?

If so, that's fine, I just wonder if any of the declarations should be used in assembler, link scripts, etc.

Reply:
@sjg20 soc.h is include in some assembly files. The SYSCLK_DEFAULT_IOSC_HZ doesn't appear in any sources. I guess it was removed some time ago.  I am removing it from mec172x soc.h